tokens/0xcfc2437916a6df165235272dbfb116687bb1a00b.yaml

```diff
@@ -0,0 +1,20 @@
+---
+addr: '0xcfc2437916a6df165235272dbfb116687bb1a00b'
+decimals: 18
+description: >-
+  A DECENTRALIZED CASHBACK PLATFORM ON THE ETHEREUM BLOCKCHAIN
+links:
+- Facebook: https://www.facebook.com/designmast
+- Facebook: https://www.facebook.com/dsplus.fork
+- Github: https://github.com/dsplus/PlusCoin
+- Telegram: https://t.me/Dsplusforkeng
+- Telegram: https://t.me/PlusCoinENG
+- Telegram: https://t.me/joinchat/DdrYdkMYRX9wWiwPgjjxqg
+- Twitter: https://twitter.com/dsplusf
+- Twitter: https://twitter.com/dsplusf?lang=ru
```

Comment:
Duplicate of the above, probably.

Reply:
Russian version, and English speaking. Should we leave both?